lumin/rollup.config.js

```diff
@@ -0,0 +1,53 @@
+import babel from 'rollup-plugin-babel';
+import resolve from 'rollup-plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import includePaths from 'rollup-plugin-includepaths';
+
+let includePathOptions = {
+  include: {},
+  paths: ['./src', '../src'],
+  external: []
+};
+
+const common = {
+  plugins: [
+    includePaths(includePathOptions),
+    babel({
+      exclude: 'node_modules/**'
+    }),
+    resolve({
+      customResolveOptions: {
+        moduleDirectory: '../node_modules'
+      }
+    }),
+    commonjs()
+  ]
+};
+
+export default [
+  // Build for MagicScript on LuminOS
+  {
+    ...common,
+    external: ['uv', 'lumin', 'ssl', 'jpeg', 'png', 'gl'],
+    input: 'src/main.js',
+    preserveModules: true,
+    output: {
+      dir: 'bin',
+      format: 'es'
+    }
+  },
+  // Build for MagicScript on Magicverse (iOS, Android)
+  {
+    ...common,
+    input: '../src/app.js',
+    external: ['react'],
+    output: {
+      globals: {
+        'react': 'React'
+      },
+      file: '../reactnative/bin/bundle.js',
```

Comment:
this should go into `lumin/bin` not reactnative.  This needs to be fixed in the template as well

Reply:
But do we really need it here? We don't have any Magicverse dependencies in this project yet

Reply:
Please update this.  It will make our lives easier having it in place correctly.